scaling_performance/cluster_limits.adoc

```diff
@@ -0,0 +1,102 @@
+[[scaling-performance-cluster-limits]]
+= Cluster Limits
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+This topic summarizes the limits for objects in {product-title}.
+
+In most cases, exceeding these thresholds results in lower overall performance.
+It does not necessarily mean that the cluster will fail over.
+
+Some of the limits represented in this topic are given for the largest possible
+cluster. For smaller clusters, the limits are proportionally lower.
+
+There are many factors that influence the stated thresholds, including the etcd
+version or storage data format.
+
+[[scaling-performance-current-cluster-limits]]
+== {product-title} 3.7 Cluster Limits
+
+[options="header",cols="3,2"]
+|===
+| Limit Type |Current Limit
+
+|Total number of all objects footnoteref:[allobjects,There is a limitation on the total number of objects on the system, as this affects etcd and its resource consumption.]
```

Comment:
This really needs to be in context of either number of namespaces (broad) or number of large applications.  I think this is almost meaningless to an end user.

Reply:
Tie this to number of namespaces in this comment.  As in "Because namespaces are a unit of tenancy, each namespace has a number of associated etcd objects.  Therefore, total scale will likely be a factor of the number of namespaces as described below" or something.

Reply:
It is definitely meaningless.  I'll vote for dropping it entirely.  Some of the content of this PR came from it's upstream equivalent, and I have already dropped many other meaningless rows that are kept upstream (including rows that just show "TBD").

Reply:
@jeremyeder Thanks. I went ahead and dropped it.